content/ai/responsible-ai/en/responsible-ai-for-llms.md

```diff
@@ -0,0 +1,206 @@
+    - Comprehensive resources
+
+* Enterprise readiness for LLM-based applications
+    - Collaboration
+    - All-in-one platform
+    - Azure Machine Learning Enterprise Readiness Solutions
+
+<img src='./docs/images/prompt-flow-view.png' width=900>
+
+### LLM-based application development lifecycle
+
+Prompt flow offers a well-defined process that facilitates the seamless development of AI applications. The lifecycle consists of the following stages:
+
+* Initialization: Identify the business use case, collect sample data, learn to build a basic prompt, and develop a flow that extends its capabilities.
+* Experimentation: Run the flow against sample data, evaluate the prompt's performance, and iterate on the flow if necessary. Continuously experiment until satisfied with the results.
+* Evaluation & Refinement: Assess the flow's performance by running it against a larger dataset, evaluate the prompt's effectiveness, and refine as needed. Proceed to the next stage if the results meet the desired criteria.
+* Production: Optimize the flow for efficiency and effectiveness, deploy it, monitor performance in a production environment, and gather usage data and feedback. Use this information to improve the flow and contribute to earlier stages for further iterations.
+
+<img src='./docs/images/prompt-flow-lifecycle.png' width=900>
+
+
+<br></br>
+# User Experience
+
+## Creating appropriate trust with the HAX Toolkit
+
+aka.ms/HAXtoolkit
+The HAX Toolkit is for teams building user-facing AI products. It helps you conceptualize what the AI system will do and how it will behave. Use it early in your design process.
+
+<img src='./docs/images/hax.png' width=900>
+
+### How to use the HAX Toolkit
+1. Familiarize yourself with the [Guidelines for Human-AI Interaction](https://www.microsoft.com/en-us/haxtoolkit/ai-guidelines/). They prescribe how an AI system should behave. Read an [overview](https://www.microsoft.com/en-us/haxtoolkit/ai-guidelines/) of the Guidelines and browse the [Design Library](https://www.microsoft.com/en-us/haxtoolkit/library/) to see examples and patterns for implementing the Guidelines.
+
+2. Bring your team together and go through the [HAX Workbook](https://www.microsoft.com/en-us/haxtoolkit/workbook/). The HAX Workbook is a discussion and planning guide. Your HAX Workbook session will result in prioritized work items for which Guidelines to implement first, and how.
+
+3. If you’re creating a product that uses natural language processing (NLP), use the [HAX Playbook](https://www.microsoft.com/en-us/haxtoolkit/playbook/) to identify how the AI system will fail so you can plan for helping users recover from inevitable failures.
+
+## Responsible AI style guide to create appropriate trust
+Kathy Walker, Mihaela Vorvoreanu
+
+Appropriate trust - users don’t over-trust and accept incorrect LLM outputs, and they don’t under-trust and reject correct LLM outputs.
+
+### What is RAI style guide?
+https://aka.ms/RAIstyleguide
```

Comment:
Not authorized to access from people outside Microsoft employee.

Reply:
Removed link until guidance from ORA.